libs/src/oracle2/services/assertions/gql/factory.ts

```diff
@@ -15,7 +16,7 @@ export const Factory =
     async function fetch({ url, chainId, address }: Config) {
       const requests = await getAssertions(url, chainId);
       return requests.map((request) =>
-        parseAssertionGraphEntity(request, chainId, address)
+        parseAssertionGraphEntity(request, chainId, address as Address)
```

Comment:
if we are gonna enforce this type, there should probably be some kind of utility "isAddress" so we dont need to cast